### PR TITLE
Health check example

### PR DESCRIPTION
**Pull request by afflerbach**

### CODE REVIEW

File: health-check/README.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+To change the check frequency, set the `interval` attribute, for example
+
+```hcl
+    beta_health {
+      expected_status = [200, 418]
+      interval = "3s"
+    }
+```
+
+Let's restart the services and filter the logs to better see what's going on:
```
Comment (malud) [inline health-check/README.md:88]: Since the watch option is enabled, Couper will handle this change.

**Review by malud — CHANGES_REQUESTED** [review]
LGTM; lets change the restart section to a reload hint?

File: health-check/README.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+  backend "fragile_backend" {
+    origin = "http://backend:8080"
+    beta_health {}
+  }
+}
+```
+
+Couper will now send `GET` requests to http://backend:8080/ every second and check
+the response status. By default, status codes of 200, 204 or 301 will be considered as healthy.
+If you want to accept other the these codes, you can set the `expected_status` attribute accordingly.
```
Comment (afflerbach) [inline health-check/README.md:78]:
```suggestion
If you want to accept other than the these codes, you can set the `expected_status` attribute accordingly.
```

File: health-check/README.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+We observe that Couper is repeatedly requesting the backend to determine its health.
+After being intially `healthy`, Couper considers the backend `failing`
+after its first "500" response. Three seconds later (`interval = "3s"`) when the status is
+still 500, the health state downgrades to `unhealthy`.
+
+The number of failing requests it takes until a backend is considered `unhealthy`
+can be configured with the `failure_threshold` attribute (defaults to 1).
+If you want the response body to contain a certain text, you can define that with
+the `expected_text` attribute. See the [reference](/avenga/couper/blob/master/docs/REFERENCE.md#health-block)
```
Comment (afflerbach) [inline health-check/README.md:106]:
```suggestion
the `expected_text` attribute. See the [reference](https://github.com/avenga/couper/blob/master/docs/REFERENCE.md#health-block)
```

Obviously GitHub does not really support relative URLs (`/` is ignored)!?

File: health-check/README.md (hunk 1 above)
Comment (afflerbach) [reply health-check/README.md]: See 96d9e8390bc7bb42fdade9195c6ba01dc015d801

**Review by malud — APPROVED** [review]

**Pull request merged**